.github/workflows/verify_consumer_pacts.yaml

```diff
@@ -72,13 +73,43 @@ on:
         type: string
 
 jobs:
+  bump-check:
+    runs-on: ubuntu-latest
+    outputs:
+      is-bump: ${{ steps.skiptest.outputs.is-bump }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Skip version bump merges
+        id: skiptest
+        uses: ./.github/actions/bump-skip
+        with:
+          event-name: ${{ github.event_name }}
+  # We only need a new version tag when this workflow is triggered by opening, updating a PR or PR merge.
+  # When triggered by a Pact Broker webhook, the provider version (GIT hash or release tag)
+  # is already included in the payload, then a new version tag wouldn't be needed.
+  #
+  regulated-tag-job:
+    needs: [ bump-check ]
+    if: ${{ needs.bump-check.outputs.is-bump == 'no' }}
+    uses: ./.github/workflows/tag.yml
+    with:
+      # The 'ref' parameter ensures that the provider version is postfixed with the HEAD commit of the PR branch,
+      # facilitating cross-referencing of a pact between Pact Broker and GitHub.
+      ref: ${{ github.head_ref || '' }}
+      # The 'dry-run' parameter prevents the new tag from being dispatched.
+      # We only need to dispatch a tag once if this is a PR merge.
+      dry-run: ${{ github.event_name == 'push' && github.ref == 'refs/heads/develop' }}
```

Comment:
Would it make sense to extract the `develop` here to use `RELEASE_BRANCHES`, defined earlier?  This might help call out an appropriate seam for setting the branch on a per-repo basis later on when this script needs to work for different repos.  I've seen repos that use both `main` and `develop` in DSP so I know this will need to change depending on context.

Reply:
I just added a RELEASE_BRANCHES envvar.